### PR TITLE
Analysis sharing

### PR DESCRIPTION
**Pull request by dennisroberts71**

### CODE REVIEW

File: services/terrain/src/terrain/util/jwt.clj [hunk 1]
```diff
@@ -64,8 +64,8 @@
 (defn validate-group-membership
   [handler allowed-groups-fn]
   (fn [request]
-    (let [allowed-groups (allowed-groups-fn)
-          actual-groups  (get-in request [:jwt-claims :org.iplantc.de:entitlement] [])]
+    (let [allowed-groups (log/spy :warn (allowed-groups-fn))
+          actual-groups  (log/spy :warn (get-in request [:jwt-claims :org.iplantc.de:entitlement] []))]
```
Comment (psarando) [inline services/terrain/src/terrain/util/jwt.clj]: Were these `warn` logging messages intentionally committed?

File: services/apps/src/apps/service/apps/jobs/sharing.clj [hunk 2]
```diff
@@ -102,10 +104,34 @@
...
 (defn- process-child-jobs
   [f job-id]
   (first (remove nil? (map f (jp/list-child-jobs job-id)))))
 
+(defn- list-job-inputs
+  [apps-client job]
+  (->> (mapv keyword (.getAppInputIds apps-client (:app-id job)))
+       (select-keys (job-params/get-job-config job))
+       vals
+       (remove string/blank?)))
```
Comment (psarando) [inline services/apps/src/apps/service/apps/jobs/sharing.clj]: Nice :+1:

File: services/terrain/src/terrain/util/jwt.clj (hunk 1 above)
Comment (dennisroberts71) [inline services/terrain/src/terrain/util/jwt.clj]: heh, nope. that was meant to be an example for DFC

File: services/apps/src/apps/service/apps/jobs/sharing.clj (hunk 2 above)
Comment (dennisroberts71) [inline services/apps/src/apps/service/apps/jobs/sharing.clj]: Thanks

**Comment by psarando** [comment]
I didn't catch any issues :shipit:

**Comment by dennisroberts71** [comment]
Cool, thanks for the review.

**Pull request merged**